tools/reviewer/job/sync/GithubClient.java

```diff
@@ -257,19 +273,25 @@ private String doRequest(RequestMethod requestMethod, String request, String req
       URL url = new URL(BASE_PATH + request);
       StringBuilder response = new StringBuilder();
       HttpURLConnection connection = (HttpURLConnection) url.openConnection();
-      connection.setRequestMethod(requestMethod.name());
+      if (requestMethod.equals(RequestMethod.PATCH)) {
+        connection.setRequestProperty("X-HTTP-Method-Override", "PATCH");
+        connection.setRequestMethod("POST");
+      } else {
+        connection.setRequestMethod(requestMethod.name());
+      }
+      connection.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
       connection.setRequestProperty(
           "Authorization",
           "Basic " + Base64.getEncoder().encodeToString((login + ":" + password).getBytes()));
-      connection.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
 
-      if (requestMethod.equals(RequestMethod.POST)) {
+      if (requestMethod.equals(RequestMethod.POST) || requestMethod.equals(RequestMethod.PATCH)) {
         connection.setDoOutput(true);
         connection.getOutputStream().write(requestData.getBytes("UTF-8"));
       }
 
       if ((connection.getResponseCode() != HTTP_OK)
-          && (connection.getResponseCode() != HTTP_CREATED)) {
+          && (connection.getResponseCode() != HTTP_CREATED)
```

Comment:
Why is `HTTP_CREATED` an error?

Reply:
This block created to simplifying debugging:
```
      if ((connection.getResponseCode() != HTTP_OK)
          && (connection.getResponseCode() != HTTP_CREATED)
          && (connection.getResponseCode() != HTTP_NO_CONTENT)) {
        StringBuilder errorResponse = new StringBuilder();
        try (BufferedReader reader =
            new BufferedReader(new InputStreamReader(connection.getErrorStream()))) {
          String line;
          while ((line = reader.readLine()) != null) {
            errorResponse.append(line);
          }
        }
        throw new IllegalStateException(
            "Error response: "
                + errorResponse
                + "\nResponse code: "
                + connection.getResponseCode()
                + ".\n"
                + connection.getResponseMessage());
      }
```
If the response code not equals HTTP_OK and HTTP_CREATED and HTTP_NO_CONTENT it is error. We read error stream to get more details about error.

Reply:
Ohh sorry, I got confused with true/false.